scripts/variantstore/wdl/GvsExtractCohortFromSampleNames.wdl

```diff
@@ -34,10 +35,22 @@ workflow GvsExtractCohortFromSampleNames {
     File? gatk_override
   }
 
+  # writing the array to a file has to be done in a task
+  # https://support.terra.bio/hc/en-us/community/posts/360071465631-write-lines-write-map-write-tsv-write-json-fail-when-run-in-a-workflow-rather-than-in-a-task
+  if (defined(cohort_sample_names_array)) {
+    call write_array_task {
+      input:
+        input_array = select_first([cohort_sample_names_array]),
+        docker = "gcr.io/google.com/cloudsdktool/cloud-sdk:305.0.0"
```

Comment:
Pinning cloud SDK versions can help avoid bugs but this particular one is ~2 years old

Reply:
Yep - I'm just taking the same version we use in other parts of the pipelines.  We may be old but at least were consistent 😆